packages/components/src/components/Tooltip/index.tsx

```diff
@@ -5,22 +5,48 @@ import Tippy, { TippyProps } from '@tippyjs/react/headless';
 import { Instance, Props as TProps } from 'tippy.js';
 import { transparentize } from 'polished';
 import { Link } from '../typography/Link';
+
 import { FONT_SIZE, FONT_WEIGHT } from '../../config/variables';
 
 type Cursor = 'inherit' | 'pointer' | 'help' | 'default';
 
 const Wrapper = styled.div``;
 
+const OpenGuideInner = styled.span`
+    float: right;
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    transition: all 0.3s ease-in-out;
+    width: 61px;
+    height: 20px;
+    border-radius: 50%;
+    cursor: pointer;
+    position: relative;
+`;
+
 const BoxDefault = styled(animated.div)<{ maxWidth: string | number }>`
-    padding: 4px 5px;
+    padding: 8px;
     background: ${props => props.theme.BG_TOOLTIP};
     color: ${props => props.theme.TYPE_WHITE};
     font-weight: ${FONT_WEIGHT.MEDIUM};
-    border-radius: 5px;
+    border-radius: 10px;
     font-size: ${FONT_SIZE.TINY};
     text-align: left;
     box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.2);
     max-width: ${props => props.maxWidth}px;
+    &:hover ${OpenGuideInner} {
+        width: 61px;
```

Comment:
this width depends on the text, right? Have you tried some more flexible solution? We will introduce another language very soon...

Reply:
sure, fixed in 1a9e24e28b84e41ea5232603340c00fc50714bbe